app/graphql/types/query_type.rb

```diff
@@ -48,12 +48,12 @@ def resources_by_figgy_ids(ids:)
   end
 
   def resources_by_bibid(bib_id:)
-    resources = query_service.custom_queries.find_by_property(property: :source_metadata_identifier, value: bib_id).select { |resource| ability.can? :discover, resource }.to_a
+    resources = query_service.custom_queries.find_by_source_metadata_identifier(source_metadata_identifier: bib_id).select { |resource| ability.can? :discover, resource }.to_a
     resources.select { |r| type_defined?(r) }
   end
 
   def resources_by_bibids(bib_ids:)
-    resources = query_service.custom_queries.find_many_by_property(property: :source_metadata_identifier, values: bib_ids)
+    resources = query_service.custom_queries.find_by_source_metadata_identifiers(source_metadata_identifiers: bib_ids)
```

Comment:
Why change the naming convention away from `find_many_by...` here? Because of length?

Reply:
I did this because `find_by_source_metadata_identifier` also returns an array of results. I can add some typedocs.

Reply:
👍🏻